test/Audio/InputSoundFile.test.cpp

```diff
@@ -15,25 +15,155 @@ TEST_CASE("[Audio] sf::InputSoundFile")
 {
     SECTION("Type traits")
     {
-        STATIC_CHECK(!std::is_default_constructible_v<sf::InputSoundFile>);
```

Comment:
Any reason to not check default constructible?

Reply:
The tests use the default constructor extensively so we don't need a type trait test for that. However, you do need a type trait to test the _absence_ of something hence that assertion being added back when this default constructor was removed.